index.js

```diff
@@ -209,6 +213,11 @@ function adapter(uri, opts, onNamespaceInitializedCallback)
 
     AMQPAdapter.prototype.__proto__ = Adapter.prototype;
 
+    AMQPAdapter.prototype.closeConnection = function ()
+    {
+        this.connection.close();
```

Comment:
is this a promise we should be returning?

Reply:
correct! Fixed